Library/Homebrew/cmd/help.rb

```diff
@@ -91,7 +91,8 @@ def command_help(path)
         line.slice(2..-1).
           sub(/^  \* /, "#{Tty.highlight}brew#{Tty.reset} ").
           gsub(/`(.*?)`/, "#{Tty.highlight}\\1#{Tty.reset}").
-          gsub(/<(.*?)>/, "#{Tty.em}\\1#{Tty.reset}")
+          gsub(/<(.*?)>/, "#{Tty.em}\\1#{Tty.reset}").
+          gsub("@hide_from_man_page", "")
```

Comment:
Stripping leading and trailing whitespace is straightforward and seems equally good.